docs/docsite/rst/user_guide/collections_using.rst

```diff
@@ -5,7 +5,7 @@
 Using collections
 *****************
 
-Collections are a distribution format for Ansible content that can include playbooks, roles, modules, and plugins. As modules move from the core Ansible repository into collections, the module documentation will move to the :ref:`collections pages <list_of_collections>`.
+Collections are a distribution format for Ansible content that can include playbooks, roles, modules, and plugins. As modules move from the core Ansible repository into collections, the module documentation will move to the `collections pages <https://docs.ansible.com/ansible/latest/user_guide/collections_using.html>`.
```

Comment:
Thanks for helping with the Ansible documentation!  One comment:

Please leave this :ref: as is. It does work and points to the collections list.  It doesn't show the full list on /devel/ yet, but it will on /latest/ once we release.

Reply:
Hi,

Thanks for pointing it out. As per my understanding refs point to another part of the same document. Bug #71254 requires fixing links resulting into 404 errors. So I should discard my change on the file 'collections_using.rst'.